src/app/public/modules/text-editor/services/text-editor-selection.service.ts

```diff
@@ -9,6 +9,15 @@ import {
   providedIn: 'root'
 })
 export class SkyTextEditorSelectionService {
+  public isElementSelected(documentEl: Document, element: HTMLElement) {
+    const selectedNode = this.getCurrentSelection(documentEl).anchorNode as HTMLElement;
+    /* istanbul ignore next */
```

Comment:
Is this "ignore" because of IE 11? If so, mind putting a comment above/below it explaining that?

Reply:
Nope. I think its just leftover from the contribution. Removed.